src/utils/parser/tests/fixtures/es6.js

```diff
@@ -0,0 +1 @@
+dispatch({ ...action, [PROMISE]: promise });
```

Comment:
I am not too sure what is happening here, where are action and promise coming from?